Amazon.QLDB.Driver/driver/AsyncQldbDriver.cs

```diff
@@ -134,29 +133,12 @@ public void Dispose()
                 }
                 catch (QldbTransactionException qte)
                 {
-                    this.driverBase.ThrowIfNoRetry(
+                    replaceDeadSession = await this.driverBase.GetShouldReplaceDeadSessionOrThrowIfNoRetryAsync(
```

Comment:
Nit - consider changing the name to IsDeadSessionOrThrowIfNoRetryAsync....

Although when put that way, it does seem like two separate things, test if dead, if not dead then check if throwing if no retry....that gets messier and we can probably go down a refactor hole.

Reply:
Lets defer this since this will be refactored out later anyways.